src/Atmos/Model/bc_moisture.jl

```diff
@@ -60,3 +60,59 @@ function atmos_moisture_normal_boundary_flux_second_order!(
     # assumes EquilMoist
     fluxᵀn.moisture.ρq_tot += nρd_q_tot
 end
+
+"""
+    BulkFormulaMoisture(fn) :: MoistureBC
+
+Calculate the net inward moisture flux across the boundary using
+the bulk formula.
+The drag coefficient is `C_q = fn_C_q(state, aux, t, normPu_int)`.
+The surface q_tot is `q_tot_sfc = fn_q_tot_sfc(state, aux, t)`.
+Return the flux (in kg m^-2 s^-1).
+"""
+struct BulkFormulaMoisture{FNX,FNM} <: MoistureBC
+    fn_C_q::FNX
+    fn_q_tot_sfc::FNM
+end
+function atmos_moisture_boundary_state!(
+    nf,
+    bc_moisture::BulkFormulaMoisture,
+    atmos,
+    args...,
+) end
+function atmos_moisture_normal_boundary_flux_second_order!(
+    nf,
+    bc_moisture::BulkFormulaMoisture,
+    atmos,
+    fluxᵀn,
+    n⁻,
+    state_sfc⁻,
+    diff_sfc⁻,
+    hyperdiff_sfc⁻,
+    aux_sfc⁻,
+    state_sfc⁺,
+    diff_sfc⁺,
+    hyperdiff_sfc⁺,
+    aux_sfc⁺,
+    bctype,
+    t,
+    state_int⁻,
+    diff_int⁻,
+    aux_int⁻,
+)
+
+    u_int⁻ = state_int⁻.ρu / state_int⁻.ρ
+    Pu_int⁻ = projection_tangential(atmos, aux_int⁻, u_int⁻)
+    normPu_int⁻ = norm(Pu_int⁻)
+    # NOTE: difference from design docs since normal points outwards
+    C_q = bc_moisture.fn_C_q(state_sfc⁻, aux_sfc⁻, t, normPu_int⁻)
```

Comment:
What's the implementation of `fn_C_q` look like?

Reply:
Right now it's just a constant. Akshay and I decided to keep it as a function for flexibility. It could be a function of the surface state (e.g. u) or time.